Testsuite failures due to oci_tools on Debian with 10.2.34

I have updated the `kiwi` package in Debian in 10.2.34 and ran the build as usual.

Starting with 10.2.34, some tests have started to fail and I have not been able to figure out yet what the problem is:

```
=================================== FAILURES ===================================
_ TestLiveImageBuilder.test_create_overlay_structure_encrypted_direct_root[erofs] _

self = <unit.builder.live_test.TestLiveImageBuilder object at 0x7f466a2dfa80>
mock_chmod = <MagicMock name='chmod' id='139940337587200'>
mock_exists = <MagicMock name='exists' id='139940337587536'>
mock_unlink = <MagicMock name='unlink' id='139940337587872'>
mock_grub_dir = <MagicMock name='get_grub_boot_directory_name' id='139940337588208'>
mock_size = <MagicMock name='SystemSize' id='139940337588544'>
mock_filesystem = <MagicMock name='new' id='139940337588880'>
mock_isofs = <MagicMock name='FileSystemIsoFs' id='139940337589216'>
mock_Iso = <MagicMock name='Iso' id='139940337589552'>
mock_tag = <MagicMock name='set_media_tag' id='139940337590224'>
mock_shutil = <MagicMock name='shutil' id='139940337590560'>
mock_Temporary = <MagicMock name='Temporary' id='139940337590896'>
mock_setup_media_loader_directory = <MagicMock name='setup_media_loader_directory' id='139940337591232'>
mock_Command_run = <MagicMock name='run' id='139940337591568'>
mock_LuksDevice = <MagicMock name='LuksDevice' id='139940337591904'>
mock_DeviceProvider = <MagicMock name='DeviceProvider' id='139940337592240'>
mock_LoopDevice = <MagicMock name='LoopDevice' id='139940337592576'>
mock_create_boot_loader_config = <MagicMock name='create_boot_loader_config' id='139940337592912'>
xml_filesystem = 'erofs'

    @mark.parametrize('xml_filesystem', ['erofs'])
    @patch('kiwi.builder.live.create_boot_loader_config')
    @patch('kiwi.builder.live.LoopDevice')
    @patch('kiwi.builder.live.DeviceProvider')
    @patch('kiwi.builder.live.LuksDevice')
    @patch('kiwi.builder.live.Command.run')
    @patch('kiwi.builder.live.IsoToolsBase.setup_media_loader_directory')
    @patch('kiwi.builder.live.Temporary')
    @patch('kiwi.builder.live.shutil')
    @patch('kiwi.builder.live.Iso.set_media_tag')
    @patch('kiwi.builder.live.Iso')
    @patch('kiwi.builder.live.FileSystemIsoFs')
    @patch('kiwi.builder.live.FileSystem.new')
    @patch('kiwi.builder.live.SystemSize')
    @patch('kiwi.builder.live.Defaults.get_grub_boot_directory_name')
    @patch('os.unlink')
    @patch('os.path.exists')
    @patch('os.chmod')
    def test_create_overlay_structure_encrypted_direct_root(
        self,
        mock_chmod,
        mock_exists,
        mock_unlink,
        mock_grub_dir,
        mock_size,
        mock_filesystem,
        mock_isofs,
        mock_Iso,
        mock_tag,
        mock_shutil,
        mock_Temporary,
        mock_setup_media_loader_directory,
        mock_Command_run,
        mock_LuksDevice,
        mock_DeviceProvider,
        mock_LoopDevice,
        mock_create_boot_loader_config,
        xml_filesystem
    ):
        temp_squashfs = Mock()
        temp_squashfs.name = 'temp-squashfs'
        temp_media_dir = Mock()
        temp_media_dir.name = 'temp_media_dir'
        tmpdir_name = [temp_squashfs, temp_media_dir]
        luks_device = mock_LuksDevice.return_value.get_device.return_value
    
        def side_effect():
            return tmpdir_name.pop()
    
        mock_Temporary.return_value.new_dir.side_effect = side_effect
        mock_size.return_value.accumulate_mbyte_file_sizes.return_value = 8192
    
        self.live_image.live_type = 'overlay'
        self.live_image.luks = 'encrypted'
        self.xml_state.build_type.get_filesystem = Mock(
            return_value=xml_filesystem
        )
    
>       self.live_image.create()

builder/live_test.py:308: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../.pybuild/cpython3_3.13/build/kiwi/builder/live.py:361: in create
    luks_size_mb = os.path.getsize(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

filename = 'kernel_name'

>   ???
E   FileNotFoundError: [Errno 2] No such file or directory: 'kernel_name'

<frozen genericpath>:86: FileNotFoundError
----------------------------- Captured stdout call -----------------------------
[ INFO    ]: 10:29:19 | Using following live ISO metadata:
[ INFO    ]: 10:29:19 | --> Application id: 0xffffffff
[ INFO    ]: 10:29:19 | --> Publisher: Custom publisher
[ INFO    ]: 10:29:19 | --> Volume id: volid
[ INFO    ]: 10:29:19 | Setting up live image bootloader configuration
[ INFO    ]: 10:29:19 | Creating live ISO boot image
[ INFO    ]: 10:29:19 | Setting up kernel file(s) and boot image in ISO boot layout
[ INFO    ]: 10:29:19 | ISO exceeds 4G size, using UDF filesystem
[ INFO    ]: 10:29:19 | Packing system into dracut live ISO type: overlay
------------------------------ Captured log call -------------------------------
INFO     kiwi:live.py:142 Using following live ISO metadata:
INFO     kiwi:live.py:143 --> Application id: 0xffffffff
INFO     kiwi:live.py:144 --> Publisher: Custom publisher
INFO     kiwi:live.py:145 --> Volume id: volid
INFO     kiwi:live.py:160 Setting up live image bootloader configuration
INFO     kiwi:live.py:188 Creating live ISO boot image
INFO     kiwi:live.py:232 Setting up kernel file(s) and boot image in ISO boot layout
INFO     kiwi:live.py:239 ISO exceeds 4G size, using UDF filesystem
INFO     kiwi:live.py:243 Packing system into dracut live ISO type: overlay
___________________________ TestOCIUmoci.test_repack ___________________________

self = <unit.oci_tools.umoci_test.TestOCIUmoci object at 0x7f4669414950>
mock_Command_run = <MagicMock name='run' id='139940333094624'>
mock_Temporary = <MagicMock name='Temporary' id='139940333097648'>

    @patch('kiwi.oci_tools.umoci.Temporary')
    @patch('kiwi.oci_tools.umoci.Command.run')
    def test_repack(self, mock_Command_run, mock_Temporary):
        mock_Temporary.return_value.new_dir.return_value.name = 'oci_root'
        oci_config = {
            'history': {
                'author': 'history author',
                'comment': 'This is a comment',
                'created_by': 'created by text'
            }
        }
        self.oci.unpack()
        self.oci.repack(oci_config)
>       assert call(
            [
                'umoci', 'repack',
                '--history.comment=This is a comment',
                '--history.created_by=created by text',
                '--history.author=history author',
                '--history.created', 'current_date',
                '--image', 'tmpdir/oci_layout:base_layer', 'oci_root'
            ]
        ) in mock_Command_run.call_args_list
E       AssertionError: assert call(['umoci', 'repack', '--history.comment=This is a comment', '--history.created_by=created by text', '--history.author=history author', '--history.created', 'current_date', '--image', 'tmpdir/oci_layout:base_layer', 'oci_root']) in [call(['umoci', 'unpack', '--image', 'tmpdir/oci_layout:base_layer', 'oci_root']),\n call(['umoci', 'repack', '--histor...e='datetime.fromtimestamp().strftime()' id='139940333095632'>, '--image', 'tmpdir/oci_layout:base_layer', 'oci_root'])]
E        +  where call(['umoci', 'repack', '--history.comment=This is a comment', '--history.created_by=created by text', '--history.author=history author', '--history.created', 'current_date', '--image', 'tmpdir/oci_layout:base_layer', 'oci_root']) = call(['umoci', 'repack', '--history.comment=This is a comment', '--history.created_by=created by text', '--history.author=history author', '--history.created', ...])
E        +  and   [call(['umoci', 'unpack', '--image', 'tmpdir/oci_layout:base_layer', 'oci_root']),\n call(['umoci', 'repack', '--histor...e='datetime.fromtimestamp().strftime()' id='139940333095632'>, '--image', 'tmpdir/oci_layout:base_layer', 'oci_root'])] = <MagicMock name='run' id='139940333094624'>.call_args_list

oci_tools/umoci_test.py:97: AssertionError
_________________________ TestOCIUmoci.test_set_config _________________________

self = <unit.oci_tools.umoci_test.TestOCIUmoci object at 0x7f466958a9c0>
mock_Command_run = <MagicMock name='run' id='139940333104704'>

    @patch('kiwi.oci_tools.umoci.Command.run')
    def test_set_config(self, mock_Command_run):
        oci_config = {
            'container_tag': 'tag',
            'entry_command': ['/bin/bash', '-x'],
            'entry_subcommand': ['ls', '-l'],
            'maintainer': 'tux',
            'user': 'root',
            'workingdir': '/root',
            'expose_ports': ['80', '42'],
            'volumes': ['/var/log', '/tmp'],
            'stopsignal': 'SIGINT',
            'environment': {'FOO': 'bar', 'PATH': '/bin'},
            'labels': {'a': 'value', 'b': 'value'},
        }
        self.oci.set_config(oci_config)
>       mock_Command_run.assert_called_once_with(
            [
                'umoci', 'config', '--author=tux', '--config.user=root',
                '--config.workingdir=/root', '--config.entrypoint=/bin/bash',
                '--config.entrypoint=-x', '--config.cmd=ls', '--config.cmd=-l',
                '--config.volume=/var/log', '--config.volume=/tmp',
                '--config.stopsignal=SIGINT',
                '--config.exposedports=80', '--config.exposedports=42',
                '--config.env=FOO=bar', '--config.env=PATH=/bin',
                '--config.label=a=value', '--config.label=b=value',
                '--no-history', '--image', 'tmpdir/oci_layout:base_layer',
                '--tag', 'tag', '--created', 'current_date'
            ]
        )

oci_tools/umoci_test.py:124: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3.13/unittest/mock.py:991: in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <MagicMock name='run' id='139940333104704'>
args = (['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/bash', ...],)
kwargs = {}
expected = call(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/...abel=b=value', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', 'current_date'])
actual = call(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/...base_layer', '--tag', 'tag', '--created', <MagicMock name='datetime.fromtimestamp().strftime()' id='139940333104368'>])
_error_message = <function NonCallableMock.assert_called_with.<locals>._error_message at 0x7f4665d9cb80>
cause = None

    def assert_called_with(self, /, *args, **kwargs):
        """assert that the last call was made with the specified arguments.
    
        Raises an AssertionError if the args and keyword args passed in are
        different to the last call to the mock."""
        if self.call_args is None:
            expected = self._format_mock_call_signature(args, kwargs)
            actual = 'not called.'
            error_message = ('expected call not found.\nExpected: %s\n  Actual: %s'
                    % (expected, actual))
            raise AssertionError(error_message)
    
        def _error_message():
            msg = self._format_mock_failure_message(args, kwargs)
            return msg
        expected = self._call_matcher(_Call((args, kwargs), two=True))
        actual = self._call_matcher(self.call_args)
        if actual != expected:
            cause = expected if isinstance(expected, Exception) else None
>           raise AssertionError(_error_message()) from cause
E           AssertionError: expected call not found.
E           Expected: run(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/bash', '--config.entrypoint=-x', '--config.cmd=ls', '--config.cmd=-l', '--config.volume=/var/log', '--config.volume=/tmp', '--config.stopsignal=SIGINT', '--config.exposedports=80', '--config.exposedports=42', '--config.env=FOO=bar', '--config.env=PATH=/bin', '--config.label=a=value', '--config.label=b=value', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', 'current_date'])
E             Actual: run(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/bash', '--config.entrypoint=-x', '--config.cmd=ls', '--config.cmd=-l', '--config.volume=/var/log', '--config.volume=/tmp', '--config.stopsignal=SIGINT', '--config.exposedports=80', '--config.exposedports=42', '--config.env=FOO=bar', '--config.env=PATH=/bin', '--config.label=a=value', '--config.label=b=value', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', <MagicMock name='datetime.fromtimestamp().strftime()' id='139940333104368'>])

/usr/lib/python3.13/unittest/mock.py:979: AssertionError
__________________ TestOCIUmoci.test_set_config_with_history ___________________

self = <unit.oci_tools.umoci_test.TestOCIUmoci object at 0x7f466958a7a0>
mock_Command_run = <MagicMock name='run' id='139940333716208'>
mock_cmd_caps = <MagicMock name='has_option_in_help' id='139940333716544'>
mock_datetime = <MagicMock name='datetime' id='139940333716880'>
mock_Temporary = <MagicMock name='Temporary' id='139940333717216'>

    @patch('kiwi.oci_tools.umoci.Temporary')
    @patch('kiwi.oci_tools.base.datetime')
    @patch('kiwi.oci_tools.umoci.CommandCapabilities.has_option_in_help')
    @patch('kiwi.oci_tools.umoci.Command.run')
    def test_set_config_with_history(
        self, mock_Command_run, mock_cmd_caps, mock_datetime, mock_Temporary
    ):
        oci_config = {
            'container_tag': 'tag',
            'entry_command': ['/bin/bash', '-x'],
            'entry_subcommand': ['ls', '-l'],
            'maintainer': 'tux',
            'user': 'root',
            'workingdir': '/root',
            'expose_ports': ['80', '42'],
            'volumes': ['/var/log', '/tmp'],
            'environment': {'FOO': 'bar', 'PATH': '/bin'},
            'labels': {'a': 'value', 'b': 'value'},
        }
        mock_Temporary.return_value.new_dir.return_value.name = 'tmpdir'
        mock_cmd_caps.return_value = False
        strftime = Mock()
        strftime.strftime = Mock(return_value='current_date')
        mock_datetime.utcnow = Mock(
            return_value=strftime
        )
        oci = OCIUmoci()
        oci.set_config(oci_config)
>       mock_Command_run.assert_called_once_with(
            [
                'umoci', 'config', '--author=tux', '--config.user=root',
                '--config.workingdir=/root', '--config.entrypoint=/bin/bash',
                '--config.entrypoint=-x', '--config.cmd=ls', '--config.cmd=-l',
                '--config.volume=/var/log', '--config.volume=/tmp',
                '--config.exposedports=80', '--config.exposedports=42',
                '--config.env=FOO=bar', '--config.env=PATH=/bin',
                '--config.label=a=value', '--config.label=b=value',
                '--image', 'tmpdir/oci_layout:base_layer',
                '--tag', 'tag', '--created', 'current_date'
            ]
        )

oci_tools/umoci_test.py:167: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3.13/unittest/mock.py:991: in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <MagicMock name='run' id='139940333716208'>
args = (['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/bash', ...],)
kwargs = {}
expected = call(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/...ue', '--config.label=b=value', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', 'current_date'])
actual = call(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/...base_layer', '--tag', 'tag', '--created', <MagicMock name='datetime.fromtimestamp().strftime()' id='139940333100000'>])
_error_message = <function NonCallableMock.assert_called_with.<locals>._error_message at 0x7f4665d9c180>
cause = None

    def assert_called_with(self, /, *args, **kwargs):
        """assert that the last call was made with the specified arguments.
    
        Raises an AssertionError if the args and keyword args passed in are
        different to the last call to the mock."""
        if self.call_args is None:
            expected = self._format_mock_call_signature(args, kwargs)
            actual = 'not called.'
            error_message = ('expected call not found.\nExpected: %s\n  Actual: %s'
                    % (expected, actual))
            raise AssertionError(error_message)
    
        def _error_message():
            msg = self._format_mock_failure_message(args, kwargs)
            return msg
        expected = self._call_matcher(_Call((args, kwargs), two=True))
        actual = self._call_matcher(self.call_args)
        if actual != expected:
            cause = expected if isinstance(expected, Exception) else None
>           raise AssertionError(_error_message()) from cause
E           AssertionError: expected call not found.
E           Expected: run(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/bash', '--config.entrypoint=-x', '--config.cmd=ls', '--config.cmd=-l', '--config.volume=/var/log', '--config.volume=/tmp', '--config.exposedports=80', '--config.exposedports=42', '--config.env=FOO=bar', '--config.env=PATH=/bin', '--config.label=a=value', '--config.label=b=value', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', 'current_date'])
E             Actual: run(['umoci', 'config', '--author=tux', '--config.user=root', '--config.workingdir=/root', '--config.entrypoint=/bin/bash', '--config.entrypoint=-x', '--config.cmd=ls', '--config.cmd=-l', '--config.volume=/var/log', '--config.volume=/tmp', '--config.exposedports=80', '--config.exposedports=42', '--config.env=FOO=bar', '--config.env=PATH=/bin', '--config.label=a=value', '--config.label=b=value', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', <MagicMock name='datetime.fromtimestamp().strftime()' id='139940333100000'>])

/usr/lib/python3.13/unittest/mock.py:979: AssertionError
__________________ TestOCIUmoci.test_set_config_dervied_image __________________

self = <unit.oci_tools.umoci_test.TestOCIUmoci object at 0x7f4669399850>
mock_Command_run = <MagicMock name='run' id='139940333725952'>

    @patch('kiwi.oci_tools.umoci.Command.run')
    def test_set_config_dervied_image(self, mock_Command_run):
        self.oci.set_config({'container_tag': 'tag'})
>       mock_Command_run.assert_called_once_with([
            'umoci', 'config', '--no-history',
            '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag',
            '--created', 'current_date'
        ])

oci_tools/umoci_test.py:184: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3.13/unittest/mock.py:991: in assert_called_once_with
    return self.assert_called_with(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <MagicMock name='run' id='139940333725952'>
args = (['umoci', 'config', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', ...],)
kwargs = {}
expected = call(['umoci', 'config', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', 'current_date'])
actual = call(['umoci', 'config', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', <MagicMock name='datetime.fromtimestamp().strftime()' id='139940333725616'>])
_error_message = <function NonCallableMock.assert_called_with.<locals>._error_message at 0x7f4665d9d620>
cause = None

    def assert_called_with(self, /, *args, **kwargs):
        """assert that the last call was made with the specified arguments.
    
        Raises an AssertionError if the args and keyword args passed in are
        different to the last call to the mock."""
        if self.call_args is None:
            expected = self._format_mock_call_signature(args, kwargs)
            actual = 'not called.'
            error_message = ('expected call not found.\nExpected: %s\n  Actual: %s'
                    % (expected, actual))
            raise AssertionError(error_message)
    
        def _error_message():
            msg = self._format_mock_failure_message(args, kwargs)
            return msg
        expected = self._call_matcher(_Call((args, kwargs), two=True))
        actual = self._call_matcher(self.call_args)
        if actual != expected:
            cause = expected if isinstance(expected, Exception) else None
>           raise AssertionError(_error_message()) from cause
E           AssertionError: expected call not found.
E           Expected: run(['umoci', 'config', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', 'current_date'])
E             Actual: run(['umoci', 'config', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', <MagicMock name='datetime.fromtimestamp().strftime()' id='139940333725616'>])

/usr/lib/python3.13/unittest/mock.py:979: AssertionError
____________ TestOCIUmoci.test_set_config_clear_inherited_commands _____________

self = <unit.oci_tools.umoci_test.TestOCIUmoci object at 0x7f4669399950>
mock_Command_run = <MagicMock name='run' id='139940333730992'>

    @patch('kiwi.oci_tools.umoci.Command.run')
    def test_set_config_clear_inherited_commands(
        self, mock_Command_run
    ):
        oci_config = {
            'container_tag': 'tag',
            'entry_command': [],
            'entry_subcommand': []
        }
        self.oci.set_config(oci_config)
>       assert mock_Command_run.call_args_list == [
            call([
                'umoci', 'config', '--clear=config.entrypoint',
                '--clear=config.cmd', '--no-history',
                '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag',
                '--created', 'current_date'
            ])
        ]
E       AssertionError: assert [call(['umoci...333730656'>])] == [call(['umoci...rrent_date'])]
E         
E         At index 0 diff: call(['umoci', 'config', '--clear=config.entrypoint', '--clear=config.cmd', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', <MagicMock name='datetime.fromtimestamp().strftime()' id='139940333730656'>]) != call(['umoci', 'config', '--clear=config.entrypoint', '--clear=config.cmd', '--no-history', '--image', 'tmpdir/oci_layout:base_layer', '--tag', 'tag', '--created', 'current_date'])
E         Use -v to get more diff

oci_tools/umoci_test.py:201: AssertionError
=========================== short test summary info ============================
FAILED builder/live_test.py::TestLiveImageBuilder::test_create_overlay_structure_encrypted_direct_root[erofs]
FAILED oci_tools/umoci_test.py::TestOCIUmoci::test_repack - AssertionError: a...
FAILED oci_tools/umoci_test.py::TestOCIUmoci::test_set_config - AssertionErro...
FAILED oci_tools/umoci_test.py::TestOCIUmoci::test_set_config_with_history - ...
FAILED oci_tools/umoci_test.py::TestOCIUmoci::test_set_config_dervied_image
FAILED oci_tools/umoci_test.py::TestOCIUmoci::test_set_config_clear_inherited_commands
================== 6 failed, 1790 passed in 550.94s (0:09:10) ==================
E: pybuild pybuild:389: test: plugin custom failed with: exit code=1: cd test/unit && pytest-3
```

Any idea?

--- a/test/unit/oci_tools/umoci_test.py
+++ b/test/unit/oci_tools/umoci_test.py
@@ -18,7 +18,7 @@ class TestOCIUmoci:
         mock_datetime.utcnow = Mock(
             return_value=strftime
         )
-        mock_datetime.fromtimestamp.return_value.strftime = Mock(
+        mock_datetime.fromtimestamp = Mock(
             return_value=strftime
         )
         self.oci = OCIUmoci()
@@ -165,6 +165,9 @@ class TestOCIUmoci:
         mock_datetime.utcnow = Mock(
             return_value=strftime
         )
+        mock_datetime.fromtimestamp = Mock(
+            return_value=strftime
+        )
         oci = OCIUmoci()
         oci.set_config(oci_config)
         mock_Command_run.assert_called_once_with(
@@ -182,7 +185,7 @@ class TestOCIUmoci:
         )
 
     @patch('kiwi.oci_tools.umoci.Command.run')
-    def test_set_config_dervied_image(self, mock_Command_run):
+    def test_set_config_derived_image(self, mock_Command_run):
         self.oci.set_config({'container_tag': 'tag'})
         mock_Command_run.assert_called_once_with([
             'umoci', 'config', '--no-history',

